pkgs/development/python-modules/jep/default.nix

```
@@ -0,0 +1,44 @@
+{ lib, python, fetchPypi, buildPythonPackage, numpy, jre, numpySupport ? true }:
+
+let
+  inherit (lib) optional optionals;
+in
+
+buildPythonPackage rec {
+  pname = "jep";
+  version = "3.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "fM0lqSoZo5FQTnZpQBkL3NTWs6hIjKSjrcjrjNWBwMs=";
+  };
+
+  buildInputs = [ jre ];
+  propagatedBuildInputs = optionals numpySupport [ numpy ];
+
+  preCheck = optional numpySupport ''
+    export PYTHONHOME="${python.withPackages(_: [ numpy ])}"
+  '';
+
+  postFixup = ''
+    cat <<EOF > ${placeholder "out"}/bin/jep
+    #!/bin/sh
+
+    if [ -z "\$1" ]
+    then
+      args="${placeholder "out"}/${python.sitePackages}/jep/console.py"
+    else
+      args=\$*
+    fi
+
+    exec ${jre}/bin/java -classpath "${placeholder "out"}/${python.sitePackages}/jep/jep-${version}.jar" -Djava.library.path="${placeholder "out"}/${python.sitePackages}/jep" jep.Run \$args
+    EOF
+  '';
+
+  meta = with lib; {
+    description = "Jep embeds CPython in Java through JNI.";
```

See https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes

```suggestion
    description = "Tool for embedding CPython in Java through JNI";
```